packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2761,14 +2762,35 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
   /**
    * Mutes/Unmutes the call.
    *
-   * @param localAudioTrack -.
+   * @param localAudioStream - The local audio stream to mute or unmute.
+   * @param muteType - Identifies if mute was triggered by system or user.
+   *
+   * @example
+   * ```javascript
+   * call.mute(localAudioStream, 'system_mute')
+   * ```
    */
-  public mute = (localAudioStream: LocalMicrophoneStream): void => {
-    if (localAudioStream) {
+  public mute = (localAudioStream: LocalMicrophoneStream, muteType?: MUTE_TYPE): void => {
+    if (!localAudioStream) {
+      log.warn(`Did not find a local stream while muting the call ${this.getCorrelationId()}.`, {
+        file: CALL_FILE,
+        method: 'mute',
+      });
+    }
+    if (muteType === MUTE_TYPE.SYSTEM) {
```

Comment:
Won't this fail if muteType is undefined?

Reply:
Why would it be undefined? We are planning to send Usermute and system mute. If nothing is passed, it will be usermute and go to else block. But for system it always has to be there

Reply:
muteType is an optional argument. That's the reason for the question.